linux_os/guide/services/usbguard/configure_usbguard_auditbackend/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,2 @@
+---
+exclude_from_count: TRUE
```

Comment:
Can you please add newline to the end of the file?

https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#all-files